cardano-wallet.cabal

```diff
@@ -81,9 +90,16 @@ library
       Cardano.Wallet.Primitive.Mnemonic
       Cardano.Wallet.Primitive.Model
       Cardano.Wallet.Primitive.Types
+      Cardano.Network
       Data.Text.Class
       Data.Quantity
       Servant.Extra.ContentTypes
+  if flag(mainnet)
+      hs-source-dirs: config/mainnet
+  if flag(staging)
+      hs-source-dirs: config/staging
+  if (!flag(staging) && !flag(mainnet))
```

Comment:
Could go for CPP within one single module, instead of having 3 different files, I guess

Reply:
Hmmm... Yes, it'll be better to have CPP pragmas within one single file, and changing the local declaration using this. Also, I'd definitely have the testnet flag too, (that is enabled by default).